src/lib/eco-router/api.ts

```diff
@@ -239,12 +268,13 @@ export async function getExactOut(
         resolve(undefined)
       })
   })
+  */
```

Comment:
remove entirely. We have git to get old code back, we don't need to commented code.

Reply:
Strong agree. Addressed in [58c241a](https://github.com/levelkdev/swapr-dapp/commit/58c241a5007dc8a549c473119897d0e3c3bbf3e2)